app/src/main/java/net/squanchy/home/HomeActivity.java

```diff
@@ -312,6 +295,41 @@ private void trackPageSelection(BottomNavigationSection section) {
         analytics.trackPageView(this, section.name());
     }
 
+    public void requestSignIn() {
+        disposeAllSubscriptions();
+        navigator.toSignInForResult(REQUEST_SIGN_IN_MAY_GOD_HAVE_MERCY_OF_OUR_SOULS);
```

Comment:
I don't really get why you have to do this, but this is obviously a hack so I'm not going to ask. 😬